config.json

```diff
@@ -183,8 +183,12 @@
         "slug": "reverse-string",
         "name": "Reverse String",
         "uuid": "c6946af0-3c2f-4f76-8b30-a1643cd5be63",
-        "practices": [],
-        "prerequisites": [],
+        "practices": [
+          "loops"
+        ],
+        "prerequisites": [
```

Comment:
Is there a concept exercise on `strings`?  If not, this exercise is going to stay locked for students.

Reply:
Yes, there is log levels (https://github.com/exercism/cpp/tree/main/exercises/concept/log-levels)
https://github.com/exercism/cpp/blob/06f46cbc72c7a6b4086e9d7f2bb054a29fe812fd/config.json#L41

Reply:
Nevermind.  I am blind as a bat, clearly.  😆